src/otx/recipe/detection/rtdetr_101.yaml

```diff
@@ -0,0 +1,115 @@
+model:
+  class_path: otx.algo.detection.rtdetr.OTXRtDetr101
+  init_args:
+    label_info: 80
+
+    optimizer:
+      class_path: torch.optim.AdamW
+      init_args:
+        lr: 0.0001
+        betas: [0.9, 0.999]
+        weight_decay: 0.0001
+
+    scheduler:
+      class_path: otx.core.schedulers.LinearWarmupSchedulerCallable
+      init_args:
+        num_warmup_steps: 5
+        main_scheduler_callable:
+          class_path: lightning.pytorch.cli.ReduceLROnPlateau
+          init_args:
+            mode: max
+            factor: 0.1
+            patience: 6
+            monitor: val/map_50
+
+engine:
+  task: DETECTION
+  device: auto
+
+callback_monitor: val/map_50
+
+data: ../_base_/data/torchvision_base.yaml
+overrides:
+  callbacks:
+    - class_path: otx.algo.callbacks.adaptive_train_scheduling.AdaptiveTrainScheduling
+      init_args:
+        max_interval: 1
+        decay: -0.025
+        min_lrschedule_patience: 3
+    - class_path: otx.algo.callbacks.adaptive_early_stopping.EarlyStoppingWithWarmup
+      init_args:
+        monitor: null
+        mode: max
+        patience: 10
+        check_on_train_epoch_end: false
+        min_delta: 0.001
+        warmup_iters: 100
+        warmup_epochs: 7
+  data:
+    task: DETECTION
+    stack_images: True
```

Comment:
Could you update it to align with other recipes by using lowercase?
https://github.com/openvinotoolkit/training_extensions/blob/debd5e05d9175bf47705abc6e24f8cdb61097bf9/src/otx/recipe/detection/yolox_tiny.yaml#L56-L65

Reply:
Done https://github.com/openvinotoolkit/training_extensions/pull/3741/commits/bda9a9d05e96151cabcbdb13320460715740b87f